.gitignore

```diff
@@ -46,3 +46,4 @@ yalc.lock
 # vercel
 .vercel
 
+analyse.html
```

Comment:
This is already on develop https://github.com/cowprotocol/cowswap/blob/7611a3deea3349bc0000ae0296f90209e452e049/.gitignore#L49

Why is it showing up as a change? 🤔